frontend/apps/app/features/organizations/actions/createOrganizations.ts

```diff
@@ -0,0 +1,55 @@
+'use server'
+
+import { createClient } from '@/libs/db/server'
+
+export type CreateOrganizationResult =
+  | { success: true; organizationId: string }
+  | { success: false; error: string }
+
+export async function createOrganization(
+  name: string,
+): Promise<CreateOrganizationResult> {
+  const supabase = await createClient()
+
+  // NOTE: Since we are using the insert method of the Supabase client, we are not performing checks such as string length validation.
+  // We should consider implementing constraints on the DB side using triggers, but this is not yet implemented as requirements are not finalized.
```

Comment:
Maybe even a Check constraint would be good 💭

Reply:
Oh! thanks! you're right